ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -18,7 +19,7 @@ public protocol CommandBarDelegate: AnyObject {
  Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItemGroups` and `trailingItemGroups` add buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
  */
 @objc(MSFCommandBar)
-open class CommandBar: UIView {
+public class CommandBar: UIView, TokenizedControlInternal {
```

Comment:
hmmm do we need to keep it open?

Reply:
https://github.com/microsoft/fluentui-apple/pull/832 Looks like we changed it in the binary size reduction effort, should we keep this change @mischreiber ?

Reply:
i would do a quick double check if any of our clients are subclassing commandbar...

Reply:
I don't think we have any subclassing business downstream!